Core/clim-core/frames.lisp

```diff
@@ -923,8 +935,9 @@ documentation produced by presentations.")
           ,@user-default-initargs)
          ,@other-options)
 
-       ,(generate-generate-panes-form
-         name menu-bar panes layouts pointer-documentation)
+       ,@(when panes
```

Comment:
This should probably be `when (or panes layouts)` otherwise frames like `demodemo` that use layouts but do not use the `:panes` option don't work.

Reply:
fixed